app/models/review_response_map.rb

```
@@ -20,4 +26,49 @@ def get_title
+
+    peer_maps.find_each do |map|
+      response = map.latest_submitted_response
+      yield response if response
+    end
+  end
+
+  # Serializes this calibration map into the JSON row expected by the
+  # assignment editor's Calibration tab.
+  # Lives here (Information Expert) — the map knows its reviewee team,
+  # its reviewer, and can ask itself for review_status.
+  def calibration_participant_json(instructor_user_id:)
```

RuboCop (1.86.1)

[warning] Unused method argument - `instructor_user_id`. You can also write as `calibration_participant_json(*)` if you want the method to accept any arguments but don't care about them.

(Lint/UnusedMethodArgument)